test/e2e/framework/util.go

```diff
@@ -675,18 +675,18 @@ func ContainerInitInvariant(older, newer runtime.Object) error {
 	if err := initContainersInvariants(newPod); err != nil {
 		return err
 	}
-	oldInit, _, _ := e2epod.Initialized(oldPod)
-	newInit, _, _ := e2epod.Initialized(newPod)
+	oldInit, _, _ := initialized(oldPod)
+	newInit, _, _ := initialized(newPod)
```

Comment:
If anything, I think we should move this to the e2epod package in order to decouple this functionality from the framework.
The method that calls e2epodInitialized is [containerInitInvariant()](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L660).
This method is used a lot in conjunction with [`framework.CheckInvariants()`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L725).

Reply:
guess for future reviews: it is ok to move a method back into the framework if it is only used by the framework (?)
/hold cancel

Reply:
Yes I think if only consumed by framework it's fine to bring it in

Reply:
But we should be careful cause it becomes really easy for someone else to export this function later and call it from somewhere else.